tumblr_client.gemspec

```diff
@@ -4,14 +4,14 @@ require File.join(File.dirname(__FILE__), 'lib/tumblr/version')
 Gem::Specification.new do |gem|
   gem.add_dependency 'faraday', '~> 0.9.0'
   gem.add_dependency 'faraday_middleware', '~> 0.9'
-  gem.add_dependency 'json'
-  gem.add_dependency 'simple_oauth'
-  gem.add_dependency 'oauth'
-  gem.add_dependency 'mime-types'
-  gem.add_development_dependency 'rake'
-  gem.add_development_dependency 'rspec'
-  gem.add_development_dependency 'webmock'
-  gem.add_development_dependency 'simplecov'
+  gem.add_dependency 'json', '~> 1.8'
```

Comment:
Hm, it doesn't seem like `json 2.x` isn't compatible with any Rubies below 2.0. Do you have any suggestions on how to deal with this? I was hoping to keep support for at least Ruby 1.9.3. :/